scripts/app

```diff
@@ -106,6 +116,11 @@ update_installed_apps() {
   echo "${updated_json}" > "${USER_FILE}"
 }
 
+# Make sure no other install script is running, if it is, wait for it to exit
+while [ $(check_if_not_already_running) == 2]; do 
+    sleep 1
```

Comment:
It would be nice to print something here like:

```
app script is already running, sleeping for 1 second...
```

Or maybe it's neater to do the log before the loop like:

```bash
if [[ "$(check_if_not_already_running)" == "2" ]]; do 
  echo "Another instance of the app script is already running, waiting for it to finish..."
  # Now do wait loop
  echo "Resuming!"
fi
```

so the logs don't get spammed.